app/src/androidTest/java/com/sdp/movemeet/view/profile/EditProfileActivityTest.java

```diff
@@ -60,17 +62,17 @@ public void onFailure(@NonNull Exception e) {
             assert(false);
         }
 
-        ActivityScenario scenario = ActivityScenario.launch(ProfileActivity.class);
+        //ActivityScenario scenario = ActivityScenario.launch(ProfileActivity.class);
```

Comment:
This can probably be removed entirely, if a test needs to launch a scenario it should do so itself I think

Reply:
Good point, that's effectively what I did. I removed this part from the before, but then the this EditeProfileActivityTest fails with following error:
`java.lang.RuntimeException: No activities found. Did you forget to launch the activity by calling getActivity() or startActivitySync or similar?
`
So I think we have to keep specify this scenario somewhere, even if it is never called, what do you think?

Reply:
Oh! I meant that the line should be at the beginning of every test instead of at the end of the setup function, so that each test can decide which activity to start from, and how to handle it. But if all tests are run in the `ProfileActivity` activity and none require special handling, leaving it there is fine

Reply:
Yes you're right, normally this kind of command has to appear at the beginning of tests rather than in the middle. I will see if I can change something!